src/model_io/codecs/src/JointElement.cpp

```diff
@@ -202,6 +202,15 @@ namespace iDynTree {
                 std::string errStr = "Joint " + m_jointName + " misses the limit tag.";
                 reportWarning("JointElement", "", errStr.c_str());
             }
+
+            if(m_dynamic_params) {
+                info.joint->setJointDynamicsType(URDFJointDynamics);
+                info.joint->setDamping(0, m_dynamic_params->damping);
+                info.joint->setStaticFriction(0, m_dynamic_params->staticFriction);
+            } else if (m_jointType == "revolute" || m_jointType == "prismatic") {
+                std::string errStr = "Joint " + m_jointName + " misses the dynamics tag.";
+                reportWarning("JointElement", "", errStr.c_str());
```

Comment:
The `dynamics` element is optional in the URDF spec (see http://wiki.ros.org/urdf/XML/joint#Elements), so I guess we should not print a warning if it is not present in the URDF?

Reply:
Good point, I removed the warning.